docs/development/adding-a-package-manager.md

```diff
@@ -14,11 +14,21 @@ Common application logic for Renovate, not specific to particular managers, usua
 
 The manager's `index.ts` file supports the following values/functions:
 
+- bumpPackageVersion (optional)
 - extractPackageFile
 - extractAllPackageFiles
 - getRangeStrategy (optional)
 - language (optional)
 - supportsLockFileMaintenance (optional)
+- updateArtifacts (optional)
+- updateDependency (optional)
+- updateLockedDependency (optional)
+
+### `bumpPackageVersion` (optional)
+
+Use this function to allow version bumps of updated packages.
+For example, increase the version of a Maven module if a package has been updated.
+Another example would be to bump the Helm chart version, if a sub chart version has been updated.
```

Comment:
```suggestion
Another example would be to bump the Helm chart version, if a subchart version has been updated.
```

The official Helm docs use `subchart` or `subcharts`, so let's follow that convention. 😄